test/integration/api/comments/delete/comments.delete.test.js

```diff
@@ -1,18 +1,18 @@
-import * as handler from '../../../../../pages/api/comments/[id]'
-
 import {
   deleteComment,
   deleteOneComment,
   findOneComment,
 } from '../../../../../lib/api/entities/comments/comments'
 
-import { apiResolver } from 'next/dist/next-server/server/api-utils'
-import { deleteActivity } from '../../../../../lib/api/entities/activity/activity'
-import fetch from 'isomorphic-unfetch'
+import {
+  deleteActivity,
+} from '../../../../../lib/api/entities/activity/activity'
```

Comment:
better not to use such paths. It's a good practice to use path with aliases like "@lib/api/entities/activity/activity"